src/utils.ts

```diff
@@ -95,7 +95,7 @@ export function loadWasm(code: string, oPath: string, nPath: string) {
     chalk.green(nPath),
   );
   code = code.replace('import.meta.url.replace(/\\.js$/, \'_bg.wasm\');', `fetch('${nPath}')`);
-  code = code.replace(`new URL('${oPath}', import.meta.url)`, `new URL('${nPath}', location.origin)`);
+  code = code.replace(`new URL('${oPath}', import\.meta\.url)`, `new URL('${nPath}', location.origin)`);
```

Comment:
For some reason `import.meta.url` autoexpanded to `file://...` *before* applying the replacement. Thus the replacement didn't happen for me.

Reply:
Somehow this doesn't happen with the vite-plugin-rsw from nodejs. Do you know what is going on?

Reply:
Did you mean that it didn’t happen? Automatically expand `file://...`?

Reply:
It did happen, and I just figured out why. tsup added this as a feature recently:

https://github.com/egoist/tsup/commit/6bcd81a43ba66c277d2357aa9a9f298cc32214b1

Reply:
basically they replace every occurrence of `import.meta.url` with `__import_meta_url`. Even inside of a string literal! So `"import.meta.url"` becomes `"__import_meta_url"`, which seems like a bug on their side.

Reply:
For now we must use `tsup < v4.8.21` or use a different bundler.